library/ziti_src.c

```diff
@@ -14,34 +14,41 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-#include <nf/ziti_src.h>
-#include <nf/ziti_log.h>
+#include <ziti/ziti_src.h>
+#include <ziti/ziti_log.h>
+#include <string.h>
 
 /**
  * Inherits from uv_lint_t and used to register as source link for `um_http`,
  * sening HTTP traffic over a Ziti connection.
  */
 typedef struct ziti_link_s {
     UV_LINK_FIELDS
-    nf_context nfc;
-    nf_connection conn;
+    ziti_context nfc;
```

Comment:
missed the variable name `nfc`

Reply:
only focused on external stuff (API, samples)

Reply:
in the samples search for `nf_` and you'll find about 20 more

Reply:
actually, try `grep -ri nf programs| grep -v config | grep --color=auto nf` found 35 total locally